Models/README.md

```diff
@@ -2,101 +2,31 @@
 
 ## Overview
 
-To run specific models, each folder contains a
-Makefile that can simply be run inside a specific model folder.
-The workflow includes running WolframScript to generate matrix elements, using Python to generate collision data, and finally processing the data with the main Python script.
+This folder contains individual model directories, each equipped with a Python script to calculate the wall velocity specific to that model. Additional files within each model directory support the computation of matrix elements and collision integrations.
 
 ## Directory Structure
 
-- `MatrixElements/` - Contains the script and data files for matrix element generation.
-- `CollisionOutput/` - Directory where collision data files are stored.
+Each model directory includes the following files and subdirectories:
+- `<nameOfModel>.py` - Contains the model definition and the commands for finding the wall velocity.
+- `<nameOfModel>Config.ini` - An optional configuration file containing model-specific configuration settings. (Note: This file is not included for the Yukawa model.)
+- `exampleCollisionDefs.py` - Defines collision generation settings specific to the model.
+- `MatrixElements/` - A subdirectory holding scripts (with .m extensions) used for generating matrix elements, as well as a .json file containing the computed matrix elements.
+- `CollisionOutput_N<spatialGridSize>/` - Subdirectories for storing collision data files for each pair of out-of-equilibrium particles, with each folder corresponding to a specific spatial grid size. 
 
+Additionally, the Models folder contains a common utility file:
 
-## Makefile Targets
+- `wallGoExampleBase.py` - This script provides a shared template for wall velocity computations. It is used by all models except the Yukawa model.
 
-### `all`
+## Running a model
 
-This is the default target which runs the `model` target.
+The examples can be run directly with e.g.
 
-### `generateMatrixElements`
-Generates matrix Elements at `MatrixElements.txt` from
-`MatrixElements/<Model>.m`
-as sometimes only the matrix elements need to be generated.
+    python3 Models/SingletStandardModel_Z2/singletStandardModelZ2.py
+    
+For models that use the common `wallGoExampleBase.py', additional command line arguments are available to (re)calculate matrix elements and collisions, and adjust the momentum grid size e.g.
 
-### `model`
-Runs only the model file
-`<model-name>.py`
-as sometimes no new matrix elements or collision integrals need to be generated.
+    python3 Models/SingletStandardModel_Z2/singletStandardModelZ2.py --recalculateMatrixElements -- recalculateCollisions --momentumGridSize 5
```

Comment:
One small typo on this line: an unexpected space after `--` and before `recalculateCollisions`.